x/accounts/defaults/multisig/account.go

```diff
@@ -387,3 +403,14 @@ func (a *Account) RegisterQueryHandlers(builder *accountstd.QueryBuilder) {
 	accountstd.RegisterQueryHandler(builder, a.QueryProposal)
 	accountstd.RegisterQueryHandler(builder, a.QueryConfig)
 }
+
+func safeAdd(nums ...uint64) (uint64, error) {
+	var sum uint64
+	for _, num := range nums {
+		if sum+num < sum {
+			return 0, errors.New("overflow")
```

Comment:
should we set it to max instead? that way tallying will see the max number not 0. 0 could signify 0 votes and passing here now

Reply:
If an error is returned we don't really care about the result, as we'll return it without continuing execution

Reply:
Makes sense. Does it make sense to check if all weight equate to less than maxuint64 when adding memebers or creating the multisig

Reply:
Q: why can't we simply ensure that total_weight <= math.Uint64Max at member lvl

Reply:
Good thinking. Checks on init/ update would be enough but tally seems not to be on the criticial path for performance so I would not mind safety over performance.

Reply:
Done, I've removed the check on tally and added the test on update